openslides_backend/action/actions/committee/committee_common_mixin.py

```diff
@@ -0,0 +1,25 @@
+from typing import Any, Dict
+
+from ....shared.exceptions import ActionException
+from ...action import Action
+
+
+class CommitteeCommonCreateUpdateMixin(Action):
+    def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
+        """Check if own committee is forwarded or received explicitly,
+        it may not be excluded by the opposite setting
+        """
+        instance = super().update_instance(instance)
+        if (
+            instance.get("forward_to_committee_ids") is None
+            or instance.get("receive_forwardings_from_committee_ids") is None
+        ):
+            return instance
+        id = instance.get("id")
+        if (id in instance.get("forward_to_committee_ids", [])) != (
+            id in instance.get("receive_forwardings_from_committee_ids", [])
+        ):
+            raise ActionException(
+                "Forwarding or receiving to/from own must be configured in both directions!"
```

Comment:
Just out of curiosity: What happens if it's only defined in one direction? Does our relation handling break?

Reply:
No, it handles both a consecutive actions